twitter-server/src/com/treode/twitter/finagle/http/package.scala

```diff
@@ -97,6 +99,52 @@ package object http {
           throw t
       }}
 
+    /** Do not use; necessary for Scala style setter. */
+	def date: TxClock  = {
+      throw new UnsupportedOperationException
+	}
+	def date_= (time: TxClock): Unit  = {
+      val dt = time.toDateTime;
+      val fmt = DateTimeFormat.forPattern("EEE, d MMM HH:mm:ss y zzz");
```

Comment:
This can be moved to out and made into a constant / singleton. That way it can be shared with lastModified below, and it can be reused from call to call rather than remade each time.